pkg/kube/configmap/configmap.go

```diff
@@ -43,6 +45,11 @@ type GetUpdateCreateDeleter interface {
 	Deleter
 }
 
+const (
+	newlineString   = "\n"
+	equalSignString = "="
```

Comment:
I was more thinking about names that give a meaning to what they represent, rather than what they are! E.g. `lineSeparator` and `fieldSeparator` or something similar! What do you think?